### ISSUE TITLE
Provide solr config via mounted folder

### ISSUE THREAD

**Issue by soniCaH** [wodby/drupal-solr#3]
In previous solr versions, I could create my configs on my machine and mount them. Since Solr 7, this seems no longer working:

```
$ docker run -v /Volumes/webdev/www/_solr/mycore:/opt/solr/server/solr/mycore -v /Volumes/webdev/www/_solr/_config_7:/opt/solr/server/solr/configsets/drupal/conf -p 8983:8983 -t wodby/drupal-solr:8-7.2
> cp: can't create directory '/opt/solr/server/solr/configsets/configsets': Permission denied
```

**Comment by csandanov** [comment]
The default user in this image is `solr`, you mount a directory with config from (most likely) a different user (with different uid). Hence, solr user has no writing permissions for this directory. You can fix this by running solr container from root user or changing permissions for this directory on your host.

**Comment by arrubiu** [comment]
Is this the only way?
I would like to enable spellcheck..

**Comment by csandanov** [comment]
My bad, actually there's was an error in entrypoint when we check for a configsets directory. Fixed now, the original command should not produce any errors. The fix included in [2.5.0 release](https://github.com/wodby/drupal-solr/releases/tag/2.5.0)
